bitnami/influxdb/README.md

```diff
@@ -129,6 +129,8 @@ As an alternative, you can use of the preset configurations for pod affinity, po
 The data is persisted by default using PVC(s). You can disable the persistence setting the `persistence.enabled` parameter to `false`.
 A default `StorageClass` is needed in the Kubernetes cluster to dynamically provision the volumes. Specify another StorageClass in the `persistence.storageClass` or set `persistence.existingClaim` if you have already existing persistent volumes to use.
 
+If you would like to define persistence settings for a backup volume that differ from the persistence settings for the database volume, you may do so under the `backup.persistence` section of the configuration. If this section is undefined, but `backup.enabled` is set to true, the backup volume will be defined using the `persistence` parameter section.
```

Comment:
Just to clarify: if someone doesn't set `backup.persistence.ownConfig: true`, and they have `persistence.existingClaim` set to some volume, should the effective value of `backup.persistence.existingClaim` be the same? That would lead to binding of the same target volume, which is the RWO problem I mentioned.

Reply:
If someone has `backup.persistence.ownConfig: false` (which is the proposed default, at least on this first version) the chart will have the very same behavior than the current version and any `backup.persistence.*` parameter will be ignored.

Any bugs present in the current version will still be there unless you explicitly set `backup.persistence.ownConfig: true`. Then, you have all the flexibility you want, right?